pyproject.toml

```diff
@@ -20,9 +20,11 @@ classifiers = [
 
 [tool.poetry.dependencies]
 python = ">=3.8,<3.11"
-ibis-framework = "^2.0.0"
+ibis-framework = "3.0.2"
```

Comment:
Does it make sense to make this `>=2,<4`?

Reply:
yes -- I had this hard-pinned because of the now-revoked versions that were misreporting `__version__`